package.json

```diff
@@ -74,7 +74,15 @@
 		"storybook": "./bin/import-wp-css-storybook.sh && start-storybook  -c ./storybook -p 6007 --ci",
 		"build-storybook": "build-storybook  -c ./storybook",
 		"changelog": "node ./bin/changelog --changelogSrcType='ZENHUB_RELEASE'",
-		"bump-version": "npm run -s install-if-deps-outdated && php ./bin/update-version.php"
+		"bump-version": "npm run -s install-if-deps-outdated && php ./bin/update-version.php",
+		"docker-dev:config": "bash ./docker/wc-admin-local-dev/config.sh",
```

Comment:
One thing that would be nice would be to put this all in one script that takes args instead, because our package json is overwhelmed with script options now its just getting unruly.